BDArmory/Control/MissileFire.cs

```diff
@@ -7787,15 +7841,12 @@ public void PointDefenseTurretFiring()
                                         if (item.Current.Vessel.Splashed && !missile.torpedo) viableTarget = false;
                                         if (interceptorsAway < maxMissilesOnTarget)
                                         {
-                                            if (viableTarget && turreted ? TargetInTurretRange(mT.turret, 7, item.Current.Vessel.CoM) : GetLaunchAuthorization(item.Current.Vessel, this, currMissile))
+                                            if (viableTarget && turreted ? TargetInTurretRange(mT.turret, mT.fireFOV, item.Current.Vessel.CoM) : GetLaunchAuthorization(item.Current.Vessel, this, currMissile))
```

Comment:
With the switch back to using `break` instead of `return`, this inner loop will need to flag to the outer loop that it also should `break` if it finds a target.